Karma/Karma/UserControlRegister.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace Karma
+{
+    public partial class UserControlRegister : UserControl
+    {
+        public UserControlRegister()
+        {
+            InitializeComponent();
+            textBox1.MaxLength = 20;
+            textBox2.MaxLength = 20;
+            textBox3.PasswordChar = '*';
+            textBox3.MaxLength = 16;
+            textBox4.PasswordChar = '*';
+            textBox4.MaxLength = 16;
+        }
+
+        private void label4_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void label2_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void label3_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void textBox2_TextChanged(object sender, EventArgs e)
+        {
+
+        }
+
+        private void textBox1_TextChanged(object sender, EventArgs e)
+        {
+
+        }
+
+        private void textBox3_TextChanged(object sender, EventArgs e)
+        {
+
+        }
+
+        private void label5_Click(object sender, EventArgs e)
+        {
+
+        }
+
+        private void textBox4_TextChanged(object sender, EventArgs e)
+        {
+
+        }
```

Comment:
These events are not used/unnecessary? Or will they be filled out later? (I guess I could also say that there are no comments)